src/util/i18nFormatting.js

```diff
@@ -1,24 +1,16 @@
-const formatFlags = {
-  EN_HANDLE_AN: 'en-handle-an',
-  CAPITALIZE: 'capitalize'
-}
+import capitalize from 'lodash/capitalize';
+
+const vowelishLetters = new Set(['a', 'e', 'i', 'o', 'u', 'h']);
+
+const formatters = {
+  'en-handle-an': val => getVariationOfAOrAn(val, false),
```

Comment:
Looks like `getVariationOfAOrAn` just takes a single parameter at this point, so we don’t need the `false`?